pkg/addons/config.go

```diff
@@ -197,4 +197,9 @@ var Addons = []*Addon{
 		set:       SetBool,
 		callbacks: []setFn{EnableOrDisableAddon},
 	},
+	{
```

Comment:
Seems this list is in alphabetical order.

Reply:
I don't think so, `csi-hostpath-driver` is after `volumesnapshots`

Reply:
Indeed. They seem to be alphabetical at the beginning, but not the last 7.

Reply:
I think the order doesn't matter as the list is sorted when it is displayed.
![image](https://user-images.githubusercontent.com/22731013/172112865-72f96c98-e66a-47ac-9587-8c77cc0db83b.png)